data/defaults.yaml

```diff
@@ -253,6 +253,18 @@ prometheus::blackbox_exporter::package_name: 'blackbox_exporter'
 prometheus::blackbox_exporter::modules: {}
 prometheus::blackbox_exporter::config_file: '/etc/blackbox-exporter.yaml'
 prometheus::blackbox_exporter::version: '0.7.0'
+prometheus::postgres_exporter::data_source_uri: 'host=/var/run/postgresql/ sslmode=disable'
+prometheus::postgres_exporter::postgres_pass: 'postgres_exporter_password'
+prometheus::postgres_exporter::postgres_user: 'postgres_exporter_username'
```

Comment:
should postgres_user and postgres_pass default to undef? Might makes it more obvious to users that they need to change it.

Reply:
agreed, 
i had to add the postgres_pass and postgres_pass to the spec test, to make the daemon start